Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM node:5.8.0
+FROM node:8.1.0
```

Comment:
This is going to require too much work for now. @Christopher-Bui and I decided we'd try to do this later and get `8.1.0` out now.